docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: "3.9"
+
+services:
+    web:
```

Comment:
I think we miss few things here for example database, possible to pass env variable to project.